lammps-tutorials.tex

```diff
@@ -231,34 +231,28 @@ \section{Introduction}
 simulations, including hybrid MPI--OpenMP parallelization
 and MPI + GPU acceleration (for a subset of its functionality).
 
-However, LAMMPS requires users to provide detailed input files that can
-be challenging to write correctly, considering the overwhelming number
-of features and options offered by LAMMPS.  While LAMMPS has
-extensive and detailed documentation, it can be difficult for new users
-to navigate.  While it focuses on describing \emph{all} available
-features, it assumes users already know which to choose.  Many of
-the advanced features are not relevant to most users, and the defaults
-usually adequate; thus, the level of detail in the manual can become
-distracting.
-
-In addition to the intrinsic complexity of LAMMPS, performing accurate
-MS requires several complex and system-specific decisions related
-to the physics to be modeled, such as selecting the thermodynamic
+LAMMPS requires users to write detailed input files, a task that
+can be particularly challenging for new users.  Although its
+documentation extensively describes all available features~\cite{lammps_docs},
+navigating it can be challenging.  Much of the information may be
+unnecessary for common use cases, and the detailed manual can
+often feel overwhelming.  Beyond the intrinsic complexity of LAMMPS, performing accurate
+MS requires several complex, system-specific decisions regarding
+the physics to be modeled, such as selecting the thermodynamic
 ensemble (e.g.~micro-canonical, grand-canonical), determining the
 simulation duration, and choosing the sets of parameters describing the
 interactions between atoms (the so-called force field)
-\cite{van2018validation}.  These choices are independent of the
-simulation software, though they can sometimes be limited by the availability of
-features in a given package.  The tutorials presented in this
-article aim to flatten the learning curve, and help users perform
-accurate and reliable MS using LAMMPS.
+\cite{wong2016good, van2018validation, prasad2018best}.  While these choices are independent of the
+simulation software, they may occasionally be constrained by the
+features available in a given package.  The tutorials in this
+article aim to ease the learning curve and guide users in
```

Comment:
```suggestion
article aim to flatten the learning curve and guide users in
```

Reply:
Sorry, but "ease the learning curve" makes no sense. You can "ease learning" but not a curve.

Reply:
You are right, my bad.